srcpkgs/flyctl/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'flyctl'
+pkgname=flyctl
+version=0.0.351
+revision=1
+build_style=gnu-makefile
```

Comment:
i think using the go build-style is better. almost all the makefile does is `go build`, and this will take care of do_install too.

you can probably run `go generate ./...` in pre_build